### PR TITLE
Reflect new Bandada API SDK version

### LINKED ISSUE #4: Update API SDK demo with the latest Bandada updates
## Description

Now the API SDK has new functions to create, update and remove groups. It would be nice to add these functions too.

### PR DESCRIPTION
**Pull request by 0xjei**
## Description
This PR updates the Bandada API SDK version from `1.1.0` to `2.2.0` adapting the code to reflect the APIs changes.

### Additional information
To minimise configuration effort, the new `createGroup` endpoint is used to create a group, removing the need for pre-creation. It then removes the invite code waiting for the API to generate one programmatically (see https://github.com/bandada-infra/bandada/issues/471). This way, only the admin's API key is required, and anyone can generate a new one after logging into the Bandada dashboard from the main page.

Note: This also removes the troubleshooting section from the README. There is no longer an expired invite code.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,4 @@
-GROUP_ID="<bandada-group-id>"
-GROUP_API_KEY="<bandada-group-api-key>"
-MEMBER_INVITE_CODE="<bandada-member-invite-code>"
+# You can generate your API key from the Bandada Dashboard home page at
+# the following link https://bandada.pse.dev/groups.
+# Just activate the API key on the dashboard, copy and paste it here.
+ADMIN_API_KEY="<bandada-admin-api-key>"
```
Comment (vplasencia) [inline .env.example]: Can you add a space at the end of this file?

**Review by vplasencia — CHANGES_REQUESTED** [review]
It looks great! Thank you very much. 

It would be nice to add the other new functions: update and remove groups. What do you think?

**Comment by 0xjei** [comment]
> It looks great! Thank you very much.
> 
> It would be nice to add the other new functions: update and remove groups. What do you think?

Yes, it makes sense and thank you for pointing this out. I will add them when adding the `createInvite` endpoint.

File: .env.example (hunk 1 above)
Comment (0xjei) [reply .env.example]: sure, one sec.

**Comment by vplasencia** [comment]
> Yes, it makes sense and thank you for pointing this out. I will add them when adding the createInvite endpoint.

Makes sense 👍

**Review by vplasencia — APPROVED** [review]

**Pull request merged**